_episodes/06-control-flow.md

```diff
@@ -83,6 +108,35 @@ Therefore she restarts her REPL.
 > {: .solution}
 {: .challenge}
 
+With the new `Trebuchet` defined with a complete `AbstractArray` interface,
+Melissa tries again to modify a counterweight by index:
+
+~~~
+trebuchet = Trebuchet(500, 0.25pi)
+~~~
+{: .language-julia}
+~~~
+2-element Trebuchet:
+ 500.0
+   0.7853981633974483
+~~~
+{: .output}
+~~~
+trebuchet[1] = 2
+~~~
+{: .language-julia}
+~~~
+2
+~~~
+{: .output}
+~~~
+print(trebuchet)
+~~~
+{: .language-julia}
+~~~
+[2.0, 0.7853981633974483]
```

Comment:
This `print` call is still superfluous
```suggestion
trebuchet
~~~
{: .language-julia}
~~~
2-element Trebuchet:
   2.0
   0.7853981633974483
```

Reply:
True. I don't often use a REPL, so it's not second-nature that `print` is superfluous within one.